apps/fz_http/lib/fz_http_web/auth/json/error_handler.ex

```diff
@@ -0,0 +1,18 @@
+defmodule FzHttpWeb.Auth.JSON.ErrorHandler do
+  @moduledoc """
+  API Error Handler module implementation for Guardian.
+  """
+  use FzHttpWeb, :controller
+  require Logger
+
+  @behaviour Guardian.Plug.ErrorHandler
+
+  @impl Guardian.Plug.ErrorHandler
+  def auth_error(conn, {type, reason}, _opts) do
+    Logger.warn("JSON.ErrorHandler.auth_error. Type: #{type}. Reason: #{reason}.")
+
+    conn
+    |> put_resp_content_type("application/json")
+    |> send_resp(401, to_string(type))
```

Comment:
Did you check if failure here raises an exception? I'm just asking because here for the socket I needed to add `Plug.Conn.halt`

Reply:
I haven't checked yet, but will keep an eye on it.